modules/ROOT/pages/install-checklist.adoc

```diff
@@ -125,9 +131,11 @@ Use the following ports to communicate with Anypoint Platform PCE:
 
 For more information, see the provider specific requirements:
 
+* https://docs.aws.amazon.com/eks/latest/userguide/eks-networking.html[Amazon EKS - Networking]
+* https://anywhere.eks.amazonaws.com/docs/clustermgmt/networking/networking-and-security/[Amazon EKS Anywhere (EKS-A) - Networking]
 * https://docs.rke2.io/install/requirements#operating-systems[Rancher Kubernetes Engine (RKE2) - Networking]
 * https://docs.openshift.com/container-platform/4.16/networking/understanding-networking.html[Red Hat OpenShift - Networking]
```

Comment:
Please remove the reference to any Openshift documentation for PCE 4.1.2 release